src/hooks/sse/usePlaylistSSE.ts

```
@@ -0,0 +1,147 @@
+        if (data?.videoCode) {
+          queryClient.setQueryData(['playlist', playlistCode], (oldData: any) => {
+            return {
+              ...oldData,
+              result: {
+                ...oldData.result,
+                nowPlayingVideoCode: data.videoCode,
+              },
+            };
+          });
+
+        const oldData: any = queryClient.getQueryData(['playlist', playlistCode]);
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)